src/jquery.uls.lcd.js

```diff
@@ -419,6 +503,18 @@
 		listen: function () {
 			var lcd = this;
 
+			this.$element.on( 'mouseenter', '.row li', function () {
+				lcd.navigationIndex = $( this ).index();
+				lcd.highlightLanguageOption();
+			} );
+
+			this.$element.on( 'mouseleave', '.row li', function () {
```

Comment:
Noticing some "mirroring" issues:

![image](https://user-images.githubusercontent.com/1685517/208079581-bd93ebb2-4e21-4343-a219-63e64c9384a9.png)

Reply:
Fixed in 8e0a9797365927f8936d010d2cdb8737b312b791